integration-cli/docker_cli_network_unix_test.go

```diff
@@ -201,7 +201,8 @@ func isNwPresent(c *check.C, name string) bool {
 	out, _ := dockerCmd(c, "network", "ls")
 	lines := strings.Split(out, "\n")
 	for i := 1; i < len(lines)-1; i++ {
```

Comment:
`range lines[1:len(lines)-2]`
But ultimately it doesn't matter because it's returning true on the first true case anyway, so false cases are fine.